internal/certdb/certdb_test.go

```diff
@@ -0,0 +1,117 @@
+package certdb_test
+
+import (
+	"log"
+	"strings"
+	"testing"
+
+	"github.com/canonical/gocert/internal/certdb"
+)
+
+func TestConnect(t *testing.T) {
+	db := new(certdb.CertificateRequests)
+	defer db.Disconnect()
```

Comment:
The linting is complaining about the error not being checked here. A way to do it in a defer clause is to pass an anonymous function:
```suggestion
	defer func() {
	    if err := db.Disconnect(); err != nil {
	        log.Errorf("Got error: %v\n", err)
	    }
	}()
```

Reply:
I know, but is it worth it to add this to everywhere where I'm not testing the disconnect? It will just grow the code in size.

Reply:
You can also add a comment to disable the linting check, I would not personally be against it in tests.

Reply:
Awesome. Done.